app/Policies/EventPolicy.php

```
@@ -0,0 +1,49 @@
+
+use App\Models\Event;
+use App\Models\User;
+
+class EventPolicy
+{
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        return ! $user->role->isBasic();
```

PHPStan (2.1.15)

Cannot call method isBasic() on int.


(method.nonObject)